commands-yml/commands/device/clipboard/get-clipboard.yml

```diff
@@ -65,11 +67,8 @@ endpoint:
     - name: session_id
       description: ID of the session to route the command to
   json_parameters:
-    - name: content
-      type: String
-      description: The actual clipboard content.
     - name: contentType
-      type: String
+      type: string
       description: The type of the content to get. Plaintext, Image, URL. Android supports only plaintext.
   response:
     - type: string
```

Comment:
the fact that the response string is base64-encoded depends on the called method. If we call the generic getClipboard then the response is encoded, but the getClipboard text shortcut performs the decoding for us and returns a plan text.